src/Codegen/Builders/CompositeBuilder.hack

```diff
@@ -0,0 +1,88 @@
+namespace Slack\GraphQL\Codegen;
+
+use namespace HH\Lib\{Vec, Keyset};
+use type Facebook\HackCodegen\{
+    CodegenClass,
+    CodegenMethod,
+    HackBuilderValues,
+    HackCodegenFactory,
+    CodegenClassConstant,
+};
+
+
+/**
+ * Builder for GraphQL objects and interfaces (has a subclass for each).
+ *
+ * The annotated Hack type should be either a class, interface, or shape.
+ */
+abstract class CompositeBuilder<TField as IFieldBuilder>
```

Comment:
This is moved from ObjectBuilder; I did `git mv` but it seems that Git lost that information after I made changes to the file :(

The only changes here are: class name, docblock, removed SUPERTYPE const